domains/certificates/Query.API/API.IntegrationTests/Shared/Migrations/UpdateTechnologyCodesMigrationTests.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Threading.Tasks;
+using API.ContractService;
+using API.Data;
+using API.IntegrationTests.Testcontainers;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql;
+using Xunit;
+
+namespace API.IntegrationTests.Shared.Migrations;
+
+public class UpdateNullTechnologyCodesMigrationTests : IAsyncDisposable
+{
+    private readonly PostgresContainer container = new();
+
+    [Fact]
+    public async Task ApplyMigration_WhenContractsHaveEmptyTechnologyCodes_UpdatesFieldsCorrectly()
```

Comment:
We should have tests for both production and consumption metering point types in the contracts